src/types.ts

```diff
@@ -20,7 +20,6 @@ export interface Holidays {
 }
 
 export interface HolidayEntry {
-  readonly region: string
```

Comment:
We no longer use `region` in the code base, but I want to keep adding it on `holidays.json` for the ease of reviewing.